packages/block-library/src/archives/index.js

```diff
@@ -20,4 +21,5 @@ export const settings = {
 	icon,
 	example: {},
 	edit,
+	deprecated,
```

Comment:
I'm not sure I understand why we need "deprecated" here at all?

Reply:
My bad there, I suggested it because I've misunderstood the error that was happening, which is caused by the block renderer endpoint instead.
I'll leave to @Addison-Stavlo to explain what's going on! 🙂

Reply:
Its not necessary... we were misinterpreting  the error and I pushed that commit so @copons could take a look. 😄 

The deprecation is not necessary and is removed, the error stems from the other comment below regarding `omit` in the `ServerSideRender` component.